### DISCUSSION TITLE
[JPTV] I Found Realabmcdn Again, But in MPEG Dash

### DISCUSSION THREAD

**Discussion by Central302** [iptv-org/community#177] [Show and tell]
I Found Realabmcdn Again, But This Time in mpd File. 
[Realabmcdn-Mpd-VLC-Test.zip](https://github.com/iptv-org/iptv/files/7575754/Realabmcdn-Mpd-VLC-Test.zip)

Also, Other Servers (JP04,NV02) Were Chenged to MPD Stream.
But These are Work Only Browser Extention.
[Realcdn-mpd-Servertest.zip](https://github.com/iptv-org/iptv/files/7575758/Realcdn-mpd-Servertest.zip)

Notice That These Server's Not Required MPD Key.

[Not Working from iOS]

**Comment by BellezaEmporium** [comment]
Works on VLC. Would have thought they would make a bigger change than this.

EDIT : JP04 works on certain ones.
JP01 works.

**Comment by BellezaEmporium** [comment]
Added channels in your list.

[fixed-list-jp.zip](https://github.com/iptv-org/iptv/files/7577210/fixed-list-jp.zip)
